### DISCUSSION TITLE
Root element <section> doesn't work with lazy loading

### DISCUSSION THREAD

**Discussion by patriktoth67** [livewire/livewire#6452] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.10

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

If the root element of my livewire component is section and not div for example, the lazy loading method doesn't seem to work. Works all right with div .

![image](https://github.com/livewire/livewire/assets/47533464/a3468489-6bae-471e-ae83-a51f1e3888fd)

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment]
Livewire components MUST have a single root element (and it's div).

reference: https://laravel-livewire.com/docs/2.x/quickstart

**Reply by patriktoth67** [reply]
I'm not 100% sure you understood the issue, but disregard this if you did.

I'm aware of it must have a single root element. My issue was that that single root element must be a \<div>, and can't be for example a \<section>. That's all, if it's intended functionality then my bad, it was just hard to diagnose since this wasn't in v2 nor v3 documentation.

**Comment by kokoshneta** [comment]
This still appears to be an issue. I had a component that used the `<body>` tag as the root element, and that threw the ‘Multiple root elements detected’ exception.

The fact that components must have a single root element seems to have been removed entirely from the docs for some reason (see #7187), but if it is indeed the case that the root element must be a `<div>`, and not a `<section>` or the `<body>` element, then that should **definitely** be mentioned in the documentation, because there is no way at all for users to know or even guess that.

**Comment by marcorieser** [comment]
Still seeing this issue on Livewire 4.

**Reply by marcorieser** [reply]
Ok, found out why this happens. When using `defer` or `lazy` on a component, Livewire renders a placeholder first. If you don't specify that, it's an empty `<div></div>`. Once the component is loaded, it tries to replace the DOM node, but you/we have used `<section></section>`.

Defining the placeholder like this, makes a root node of `<section></section>` work.
```php
  public function placeholder() {
      return '<section></section>';
  }
```
https://livewire.laravel.com/docs/4.x/attribute-defer#rendering-placeholders
https://livewire.laravel.com/docs/4.x/attribute-lazy#rendering-placeholders
